pkg/plugins/golang/v3/scaffolds/internal/templates/gowork.go

```diff
@@ -0,0 +1,131 @@
+/*
```

Comment:
I think the idea of fixing https://github.com/kubernetes-sigs/kubebuilder/issues/2627 would see if we can help allow them to create a go. mod in this directory but we do not need to provide a default scaffold and officially support. Just to see what changes we need to do in the tools to not block their action. 

But please, share your thoughts I might missing something here.

Reply:
You can always manually create go.mod and go.work files and do the scaffold I am doing here but then the `create api` commands will fail since it will try to use `go mod tidy` in the PostProcessing. This will only work if you issue replace directives in the root `go.mod` => This is what I wanted to avoid since it makes it really hard to maintain.